nucypher/blockchain/eth/constants.py

```diff
@@ -19,3 +19,7 @@
 
 ETH_ADDRESS_BYTE_LENGTH = 20
 ETH_ADDRESS_STR_LENGTH = 40
+
+
+# Solidity
+SOLIDITY_MAX_UINT16 = 65535
```

Comment:
Based on subsequent usage, I would just remove the `SOLIDITY_` prefix
```suggestion
MAX_UINT16 = 65535
```

OR 

```suggestion
MAX_PERIOD_VALUE = 65535
```

Reply:
✔️